GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts

```diff
@@ -120,44 +148,204 @@ namespace gdjs {
       if (displayHitboxesAndSomePoints) {
         for (let i = 0; i < instances.length; i++) {
           const object = instances[i];
+          const id = object.id;
+          const layer = this._runtimeScene.getLayer(object.getLayer());
+          const cameraTopLeftX =
+            layer.getCameraX() - layer.getCameraWidth() / 2;
+          const cameraTopLeftY =
+            layer.getCameraY() - layer.getCameraHeight() / 2;
+
+          if (
+            (!object.isVisible() || !layer.isVisible()) &&
+            !showHiddenInstances
+          ) {
+            continue;
+          }
           const cameraCoords = layersCameraCoordinates[object.getLayer()];
           const rendererObject = object.getRendererObject();
           if (!cameraCoords || !rendererObject) {
             continue;
           }
 
+          // Create the structure to store the points in memory
+          if (!this._debugDrawRenderedObjectsPoints[id]) {
+            this._debugDrawRenderedObjectsPoints[id] = {
+              wasRendered: true,
+              points: { x: 0, y: 0 },
```

Comment:
This seems false, points is not a single point, it's a map associating points name to their associated pixi texts.